src/Skoruba.IdentityServer4.STS.Identity/Program.cs

```diff
@@ -8,14 +9,21 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            CreateWebHostBuilder(args)
-                .UseSerilog()
-                .Build().Run();
+            CreateHostBuilder(args).Build().Run();
         }
 
-        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .UseIISIntegration()
-                .UseStartup<Startup>();
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureLogging((hostingContext, logging) => {
+                    logging.ClearProviders();
+                    var logger = new LoggerConfiguration().ReadFrom.Configuration(hostingContext.Configuration).CreateLogger();
+                    logging.AddSerilog(logger);
+                })
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.ConfigureKestrel(options => options.AddServerHeader = false);
+                    webBuilder.UseStartup<Startup>();
+                    webBuilder.UseSerilog();
```

Comment:
@fatalistt 

You don't need _UseSerilog_ on _IWebHostBuilder_

Reply:
Sorry for a long anwer, I was busy at work.
Yes, you're right, `IWebHostBuilder.UseSerilog` is redunant.

Reply:
Can you remove this line before final merge, please? :-)
Thanks

Reply:
Already done that :)